packages/interactivity/src/proxies/signals.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import {
+	computed,
+	signal,
+	batch,
+	type Signal,
+	type ReadonlySignal,
+} from '@preact/signals-core';
```

Comment:
Could we import everything from `@preact/signals` to avoid mixing packages?

```suggestion
} from '@preact/signals';
```

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/62734/commits/9fb783cfe5fca6ebb3ab2b939253a90f34eb4bff.